app/src/server/scripts/removeDuplicateInvoices.ts

```diff
@@ -0,0 +1,35 @@
+import { prisma } from "../db";
```

Comment:
add a check that we only delete canceled invoices

Reply:
Added!